website/src/css/custom.css

```
@@ -79,6 +79,14 @@ html {
 }
 
 body {
+  background: var(--smocket-bg);
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)